modules/gitops-storing-and-retrieving-argo-cd-logs.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * cicd/gitops/viewing-argo-cd-logs-using-the-kibana-dashboard.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-storing-and-retrieving-argo-cd-logs_{context}"]
+= Storing and retrieving Argo CD logs
+
+You can use the Kibana dashboard to store and retrieve Argo CD logs.
+
+.Procedure
+
+. In the {product-title} web console, navigate to the {rh-app-icon} menu -> *Observability* -> *Logging*. The Kibana dashboard is opened in a new window.
+
+. Create an index pattern.
+
+.. To display all the indeces, define the index pattern as "***" . Click *Next step* to continue.
+
+.. Select *@timestamp* for _Time Filter field name_.
+
+.. Click *Create index pattern* and wait until the index pattern is created.
```

Comment:
```suggestion
.. Click *Create index pattern*.
```
- Consider cutting out the second part of the sentence. It's self-explanatory that clicking **Create index pattern** will create an index pattern.